server/src/storage/storage.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { StorageService } from './storage.service';
+import { LocalStorageService } from './providers/local/local.storage.service';
+
+@Module({
+  providers: [StorageService, LocalStorageService],
+  exports: [StorageService, LocalStorageService],
```

Comment:
In the current implementation, `LocalStorageService` is only injected into StorageCore service so this module doesn't need to export it.
```suggestion
  exports: [StorageService],
```